client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -1,152 +1,217 @@
-						<div className="plugins-browser-item__title">{ plugin.name }</div>
-						{ variant === PluginsBrowserElementVariant.Extended && (
+	const classNames = classnames( 'plugins-browser-item', variant );
+	return (
+		<li className={ classNames }>
+			<a
+				href={ pluginLink }
+				className="plugins-browser-item__link"
+				onClick={ trackPluginLinkClick }
+			>
+				<div className="plugins-browser-item__info">
+					<PluginIcon size={ iconSize } image={ plugin.icon } isPlaceholder={ isPlaceholder } />
+					<div className="plugins-browser-item__title">{ plugin.name }</div>
+					{ variant === PluginsBrowserElementVariant.Extended && (
+						<>
 							<div className="plugins-browser-item__author">
 								{ translate( 'by ' ) }
 								<span className="plugins-browser-item__author-name">{ plugin.author_name }</span>
 							</div>
+							{ dateFromNow && (
+								<div className="plugins-browser-item__last-updated">
+									{ translate( 'Last updated ' ) }
+									<span className="plugins-browser-item__last-updated-value">{ dateFromNow }</span>
+								</div>
+							) }
+						</>
+					) }
+					<div className="plugins-browser-item__description">{ plugin.short_description }</div>
+				</div>
+				{ isUntestedVersion && (
+					<div className="plugins-browser-item__untested-notice">
+						<Icon size={ 20 } icon={ info } />
+						<span className="plugins-browser-item__untested-text">
+							{ translate( 'Untested with your version of WordPress' ) }
+						</span>
+					</div>
+				) }
+				<div className="plugins-browser-item__footer">
+					{ variant === PluginsBrowserElementVariant.Extended && (
+						<InstalledInOrPricing
+							sitesWithPlugin={ sitesWithPlugin }
+							isWpcomPreinstalled={ isWpcomPreinstalled }
+						/>
+					) }
+					<div className="plugins-browser-item__additional-info">
+						{ !! plugin.rating && (
+							<div className="plugins-browser-item__ratings">
+								<PluginRatings
+									rating={ plugin.rating }
+									inlineNumRatings={
+										plugin.rating && Number.isInteger( plugin.num_ratings )
+											? plugin.num_ratings.toLocaleString( getLocaleSlug() )
+											: null
+									}
+									hideRatingValue
+								/>
+							</div>
+						) }
+						{ !! plugin.active_installs && (
+							<div className="plugins-browser-item__active-installs">
+								<span className="plugins-browser-item__active-installs-value">{ `${ formatNumberMetric(
+									plugin.active_installs
+								) }${ plugin.active_installs > 1000 ? '+' : '' }` }</span>
+								{ translate( ' Active Installs' ) }
+							</div>
 						) }
-						<div className="plugins-browser-item__description">{ plugin.short_description }</div>
 					</div>
-					{ variant === PluginsBrowserElementVariant.Extended && this.renderInstalledInOrPricing() }
-				</a>
-				{ this.renderUpgradeButton() }
-			</li>
+				</div>
+			</a>
+			<UpgradeButton plugin={ plugin } />
```

Comment:
It seems we can safely entirely remove the `UpgradeButton` since it was used for features not includes in your plan https://github.com/Automattic/wp-calypso/pull/18323/commits/4db1944654ecf1c6edf6136e6475b9418b5a2a34 and not for plugins.

Reply:
done 8284966f3a808426e9a78433dd75c7ee74df16d7